torch_train.py

```diff
@@ -0,0 +1,42 @@
+import shutil
```

Comment:
Seems to be identical to tf_train with the exception of the training class used, might make more sense to just accept an argument for the desired training type and use that in a branching statement?

Reply:
Good point, the manual training scripts leave a lot to be desired in general. They were sort of an afterthought once I shifted into the mode to get the rest of the library to the point where we could reference it in the API and the pipeline.

I think maybe a single manual_train.py having a default args instance, but that also accepts command line params to override arg values (and model type) might be best. We can update the default args to track with the high scoring model at any given time. I'll try to drop this into Zenhub for the next sprint

Reply:
Sounds great!